### PR TITLE
Minor pre-release warning fixes

### PR DESCRIPTION
**Pull request by QuLogic**
Some minor cleanup that caused some warnings while testing out the release candidate.

### CODE REVIEW

File: obspy/imaging/waveform.py [hunk 1]
```diff
@@ -500,7 +500,7 @@ def plot_day(self, *args, **kwargs):
         if "min_magnitude" in events:
             try:
                 from obspy.clients.fdsn import Client
-                c = Client("NERIES")
+                c = Client("EMSC")
```
Comment (krischer) [inline obspy/imaging/waveform.py]: I guess this should become a parameter in the long run. Also `ISC` might be the correct resource but it likely returns a metric ton of events which might be too much for a plot.

**Comment by krischer** [comment]
Looks good to me. I'll merge once CI passes.

File: obspy/imaging/waveform.py (hunk 1 above)
Comment (QuLogic) [inline obspy/imaging/waveform.py]: Yep, that's #887.

Comment (krischer) [inline obspy/imaging/waveform.py]: We have too many issues ;-)

**Pull request merged**